### PR TITLE
[nnfeapi] Add one-op test for InstanceNorm

### PR DESCRIPTION
**Pull request by wateret**
Add basic one-op test for InstanceNorm.

ONE-DCO-1.0-Signed-off-by: Hanjoung Lee <hanjoung.lee@samsung.com>

### CODE REVIEW

**Comment by wateret** [comment]
I expect this to improve test coverage widely - from loader to kernel.

File: tests/nnfw_api/src/one_op_tests/InstanceNorm.cc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  int out = cgen.addTensor({{1, 2, 2, 1}, circle::TensorType::TensorType_FLOAT32});
+
+  cgen.addOperatorInstanceNorm({{in, beta, gamma}, {out}}, 0,
+                               static_cast<circle::ActivationFunctionType>(128) /* Invalid value*/);
+  cgen.setInputsAndOutputs({in}, {out});
+
+  _context = std::make_unique<GenModelTestContext>(cgen.finish());
+  _context->expectFailModelLoad();
+
+  SUCCEED();
```
Comment (ragmani) [inline tests/nnfw_api/src/one_op_tests/InstanceNorm.cc:56]: Is this test positive?

Comment (ragmani) [reply tests/nnfw_api/src/one_op_tests/InstanceNorm.cc:56]: Sorry I misunderstood.

**Review by ragmani — APPROVED** [review]
LGTM

**Review by Hyunjun85 — APPROVED** [review]

**Comment by juitem** [comment]
LGTM

**Review by juitem — APPROVED** [review]

**Pull request merged**